test/unit/app/models/configurable_content_blocks/default_date_test.rb

```diff
@@ -47,11 +51,25 @@ class ConfigurableContentBlocks::DefaultDateRenderingTest < ActionView::TestCase
       schema: @schema["properties"]["test_attribute"],
       content: @block_content["test_attribute"],
       path: Path.new.push("test_attribute"),
+      translated_content: nil,
     }
 
-    assert_dom "input[name=\"edition[block_content][test_attribute(3i)]\"][value=\"#{@date.day}\"]"
-    assert_dom "input[name=\"edition[block_content][test_attribute(2i)]\"][value=\"#{@date.month}\"]"
-    assert_dom "input[name=\"edition[block_content][test_attribute(1i)]\"][value=\"#{@date.year}\"]"
+    assert_dom "input[name=\"edition[block_content][test_attribute][3]\"][value=\"#{@date.day}\"]"
```

Comment:
I'd fix this up to the previous commit (as presumably it started failing from commit 2)